universe/jupyterlab/1/config.json

```diff
@@ -119,6 +119,16 @@
           "type": "string",
           "default": "/logmeout"
         },
+        "oidc_authorization_params": {
+          "description": "OIDC Authorization Parameters.",
```

Comment:
Just as an internal reminder: We should add examples/links to the documentation here service guide...

Reply:
@mzarrabi Please track ^^^ for update to the Service Docs.